Doc/whatsnew/3.13.rst

```diff
@@ -295,6 +295,13 @@ venv
   (using ``--without-scm-ignore-files``). (Contributed by Brett Cannon in
   :gh:`108125`.)
 
+warnings
+--------
+
+* The new :func:`warnings.deprecated` decorator provides a way to communicate
+  deprecations to :term:`static type checkers <static type checker>`. See
```

Comment:
I think this should also mention that it's a nice, ergonomic way to add deprecations at runtime